specs/lightclient/beacon-chain.md

```diff
@@ -155,6 +157,11 @@ class BeaconState(Container):
     # Light client sync committees
     current_sync_committee: SyncCommittee
     next_sync_committee: SyncCommittee
+    # How many inactivity leak epochs have there been in the previous penalty period?
+    leak_epoch_counter: uint64
+    # Increases when the validator misses epochs in an inactivity leak, decreases when the validator
```

Comment:
The problem here is that if there is `leak-1` and the validator gets a heaving leak epoch counter, then some time in the future (after the vlaidator became a good citizen) there is a `leak-2` the validator starts with a pretty big leak and has to climb back out (if the leak continues)

Alternatives
1. increase/decrease `inactive_score` during all epochs (instead of just leak). This allows a validator to clean up any issues from a past leak and also if a validator has been offline for a long time, the validator (who doesn't have a good chance of coming back online during a leak) immediately gets penalized highly. Main problem with this is that it could insta-burn the whole validator account... 
2. decrease during leak but allow to climb back out on non-leak. This has the advantage of not having your performance in one leak necessarily affect a leak that is completed disjoint. It also ensures that validators don't get immediately destroyed if they've been offline during a healthy period prior to the leak. The mean disadvantage I see is that it could make the leak slower in times where the chain is going in an out of leak period and validators are online then offline repeatedly.

Given how I see the tradeoffs, I have a slight preference for (2)

Reply:
This also might help counter-act the leak score bias to be a bit more fair. It takes more epochs to pull yourself out of the leak penalty state than to enter but you can pull yourself out in healthy epochs too